testdata/go/v3/memcached-operator/controllers/memcached_controller.go

```diff
@@ -180,6 +180,7 @@ func (r *MemcachedReconciler) deploymentForMemcached(m *cachev1alpha1.Memcached)
 									"ALL",
 								},
 							},
+							RunAsUser: &[]int64{1000}[0],
```

Comment:
We also need to improve the checks like we did in the PR in kubebuilder: https://github.com/kubernetes-sigs/kubebuilder/pull/2772

See that we are verifying if all is restricted.
We might need to enforce that the ns where the pod Memcached sample will be installed is restricted.

Reply:
That's a good point. It may be a good idea to do this in a follow-up as well.